dist/dialog-mini/ds4/dialog-mini.css

```diff
@@ -0,0 +1,141 @@
+.dialog {
+  --dialog-mask-background-color: #333;
+  --dialog-window-background-color: #fff;
+  --dialog-back-close-foreground-color: #333;
+}
+.dialog-mini[role="dialog"] {
+  background-color: rgba(51, 51, 51, 0.7);
+  bottom: 0;
+  left: 0;
+  -webkit-overflow-scrolling: touch;
+  overflow-y: auto;
+  position: fixed;
+  right: 0;
+  top: 0;
+  will-change: background-color;
+  z-index: 100000;
+}
+.dialog-mini--no-mask[role="dialog"] {
```

Comment:
Do we still need this no mask modifier? Is it a leftover from the really, really old markup we had? Or still used?

Reply:
Oops. comment was meant to be on the dialog.less file, not mini, but same applies.